### ISSUE TITLE
[Feedback] partitioning_column - no reference found

### ISSUE THREAD

**Issue by weberth** [timescale/docs#3168]
### Is it easy to find the information you need?

Yes

### Are the instructions clear?

No

### How could we improve the Timescale documentation site?

Regarding this doc: https://docs.timescale.com/use-timescale/latest/hypertables/hypertables-and-unique-indexes/
_Any space-partitioning columns. Space partitions are optional and not included in every hypertable. You have a space-partitioning column if you specified the **partitioning_column** parameter when you creates your hypertable._

I can't find some useful doc for partitioning_column. Neither in the doc-search nor in the reference https://docs.timescale.com/api/latest/hypertable/create_hypertable/ .

**Comment by jonatas** [comment]
Good point @weberth! That's a left over as partitioning_column probably belongs to the `dimension` that you can use. It's related to an [old syntax](https://docs.timescale.com/api/latest/hypertable/create_hypertable_old/).

**Comment by jonatas** [comment]
@mkindahl any recommendation how to follow up the docs in this page considering that we have `dimension`?
